docs/common-bugs.rst

```diff
@@ -187,6 +187,14 @@ Failing hardware
 :Solution: It looks like a hardware failure. Please, check your hardware.
 :Example: `rhbz#1685047 <https://bugzilla.redhat.com/show_bug.cgi?id=1685047>`_
 
+LVM on disks with inconsistent sector size
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Detection: Error with message mentioning "inconsistent sector size".
+:Solution: LVM is now demanding that all disks have consistent sector size, otherwise they can't
+    be used together. Please look on
+    `rhbz#1669751 <https://bugzilla.redhat.com/show_bug.cgi?id=1669751>`_ for more info.
+
```

Comment:
I would mention the bug https://bugzilla.redhat.com/show_bug.cgi?id=1754683 as an example. It is a public duplicate of the Blivet bug that was dealing with this problem.

Reply:
I wasn't able to find it. Thanks!